src/components/CustomExtendedDataField/CustomExtendedDataField.js

```diff
@@ -29,8 +31,16 @@ const CustomFieldEnum = props => {
     intl.formatMessage({ id: 'CustomExtendedDataField.placeholderSingleSelect' });
   const filterOptions = createFilterOptions(enumOptions);
 
+  const label = getLabel(fieldConfig);
+
   return filterOptions ? (
-    <FieldSelect className={css.customField} name={name} id={name} label={label} {...validateMaybe}>
+    <FieldSelect
+      className={css.customField}
+      name={name}
+      id={name}
+      label={label || fieldLabel}
```

Comment:
I think this _fieldLabel_ is not defined anywhere.

Reply:
True, removed now!